EXILED/Exiled.API/Features/Items/Throwable.cs

```diff
@@ -62,6 +62,11 @@ public override Vector3 Scale
         /// </summary>
         public Projectile Projectile { get; }
 
+        /// <summary>
+        /// Gets or sets how long the fuse will last.
+        /// </summary>
+        public float FuseTime { get; set; }
```

Comment:
Why not make this virtual?

Reply:
I wasn't sure if it's would cause breaking change with the one above